sous-board-2019 (#83)
sous-chefs/website: content/docs/sous-board-2019.md

---
title: 'Community Leaders 2019'
date: "2019-10-10T00:00:00-00:00"
---

> Thank you all for voting in the Sous-Chefs Community Leaders, voting has now closed and the results are in

Reviewer: ```suggestion
Thank you all for voting on the board of Sous-Chefs Community Leaders. Voting has now closed and the results are in.
```

## Farewells

> We would like to with our fond farewells to the members of the board who have chosen to step down, Dan and Joseph have been instrmental in sous-chefs and will be missed, some of their great achievements are:

Reviewer: ```suggestion
We would like to wish our fond farewells to the members of the board who have chosen to step down: Dan and Joseph have been instrumental in Sous-Chefs and will be missed. Some of their great achievements are:
```

> - the direction and rewrite of resource driven cookbooks, that has been great uptake in the community.

Reviewer: ```suggestion
- The direction and rewrite of resource driven cookbooks that has been a great uptake in the community
```

- CI improvements that make releasing cookbooks easy and safe
- Rewriting the Apache2 cookbook

We wish them all the best and hope that they remain active members of the community

## Board of 2019

We would like to introduce and welcome the Sous-Chefs board:

### Tim Smith

> [Tas50](https://github.com/Tas50), is coninuting on in his Leadership position within Sous-Chef, Tim has been pivotal to Sous-Chefs and aiding us in writing better cookbooks along with some amazing work for chef which has allowed us to automate more testing and test fixes than ever before. We are greatful for his continuing work with us

Reviewer: ```suggestion
[Tas50](https://github.com/Tas50) is continuing on in his Leadership position within Sous-Chef. Tim has been pivotal to Sous-Chefs and has aided in writing better cookbooks along with some amazing work for Chef, which has allowed us to automate more testing and test fixes than ever before. We are grateful for his continuing work! with us
```

### John Roesler

> [John](https://github.com/JohnRoesler), is new to the board this year and has been an active member of the Sous-Chefs community and in the last year has been working a lot on [HaProxy cookbook](https://github.com/sous-chefs/haproxy/) along with all the other cookbooks we mange, we are really excited to have him join us

Reviewer: ```suggestion
[John](https://github.com/JohnRoesler) is new to the board this year and has been an active member of the Sous-Chefs community. In the last year, he has been working a lot on the [HaProxy cookbook](https://github.com/sous-chefs/haproxy/) along with all the other cookbooks we mange. We are really excited to have him join us!
```

### Jason Field

> [Xorima](https://github.com/Xorima), is new to the board this year having been active in Sous-Chefs and awarded Awesome Community Chef at ChefConf London. He has predominantly focused on helping us move to the new CircleCI build pipeline and ensuring all our cookbooks can pass their tests, we look forward to what he has to bring to the board

Reviewer: ```suggestion
[Xorima](https://github.com/Xorima) is new to the board this year and has been active in Sous-Chefs over the past year and he was awarded Awesome Community Chef at ChefConf London. He has been predominantly focused on helping us move to the new CircleCI build pipeline and has worked to ensure all our cookbooks can pass their tests. We look forward to what he has to bring to the board!
```

We look forward to the work they will do to help the community grown and thrive moving forward

## What does the board do

The members of the Sous-Chefs board are responsible for:

- Growing the Sous-Chefs Open Source Community

> - keeping things ticking along, supporting shipping some of the most used cookbooks in the Chef community.

Reviewer: ```suggestion
- Keeping things ticking along, supporting shipping some of the most used cookbooks in the Chef community
```

> - Mentor and guide others to produce great cookbooks from ground up.

Reviewer: ```suggestion
- Mentor and guide others to produce great cookbooks from the ground up
```

## Get Involved

> If you still don't feel like nominating yourself for the vote, there are plenty of other ways to get involved like opening pull requests, answering or even asking, questions on Slack.

Reviewer: ```suggestion
If you still don't feel like nominating yourself for the board vote, there are plenty of other ways to get involved like: opening pull requests, answering and asking questions on Slack, and opening or commenting on issues.
```
